src/main/java/org/jitsi/jicofo/JitsiMeetConferenceImpl.java

```diff
@@ -2115,11 +2120,23 @@ public MuteResult handleMuteRequest(Jid muterJid, Jid toBeMutedJid, boolean doMu
             return MuteResult.ERROR;
         }
 
-        // do not allow unmuting other participants even for the moderator
-        if (!doMute && !muterJid.equals(toBeMutedJid))
+        // process unmuting
+        if (!doMute)
         {
-            logger.warn("Unmute now allowed, mutedJid=" + muterJid + ", toBeMutedJid=" + toBeMutedJid);
-            return MuteResult.NOT_ALLOWED;
+            // do not allow unmuting other participants even for the moderator
+            if (muterJid != null && !muterJid.equals(toBeMutedJid))
```

Comment:
I think this should be `muterJid == null || !muterJir.equals(toBeMutedJid)` to allow unmute only when a participant unmutes themselves.